tests/integ/modin/frame/test_dropna.py

```diff
@@ -84,7 +84,7 @@ def test_axis_1_not_implemented(test_dropna_df):
         df.dropna(axis="columns")
 
 
-@sql_count_checker(query_count=1)
```

Comment:
you will also need to fix the expected error for this test

Reply:
it still raises the same errors, none of them are run on axis==1

Reply:
fixed it